### DISCUSSION TITLE
"Unecpected API error" when trying to configure the integration.

### DISCUSSION THREAD

**Discussion by henkha** [pypolestar/polestar_api#267] [Q&A] [answered]
I had the integration running for a few days. However, it stopped working the other day and I tried to delete the configuration and start over. However, after entering my PolestarID, Password and VIN, I receiv an error message saying that an "unexpected API error" occurred.

![image](https://github.com/user-attachments/assets/a65e2e23-00e6-4f75-a506-815dec1039d9)

**Comment by BJReplay** [comment, marked as answer]
Polestar changed their authentication approach the other day.  You may not have been prompted to update to the latest release by HACS - if you go to HACS, select the Polestar API, select the three dots near the top right, and choose Update information (it might say Informationen zum Update for you) HACS should pick up the latest release https://github.com/pypolestar/polestar_api/releases/tag/1.11.0 which has a fix for the change in authentication.

Download the update, restart HA, and you might find that this might fix the problem for you.

![image](https://github.com/user-attachments/assets/8821255b-ca0f-4b51-a66f-732d2572d4b4)

**Comment by henkha** [comment]
Yes that helped and solved my problem, thank you!
